scrapers/bruceandmorgan.yml

```diff
@@ -0,0 +1,20 @@
+name: Bruce and Morgan
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://bruceandmorgan.net/scene/
```

Comment:
Protocol should be dropped from the URL so that it is just `bruceandmorgan.net/scene/`